libraries/AP_InertialSensor/AP_InertialSensor_Invensensev3.h

```diff
@@ -75,6 +75,11 @@ class AP_InertialSensor_Invensensev3 : public AP_InertialSensor_Backend
     bool accumulate_samples(const struct FIFOData *data, uint8_t n_samples);
     bool accumulate_highres_samples(const struct FIFODataHighRes *data, uint8_t n_samples);
 
+    // get the gyro backend rate in Hz at which the FIFO is being read
+    uint16_t get_gyro_backend_rate_hz() const override {
```

Comment:
I think we should take the opportunity to come up with a better name than "backend". Perhaps "get_gyro_read_rate_hz()" at least conveys what is going on and why it might be important.

Reply:
That needs to be much larger and a separate change, I don't want to widen the scope of this PR.

Reply:
you could just remove the word "backend" from the method name maybe?